fdbclient/ServerKnobs.cpp

```diff
@@ -377,6 +377,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( REPLACE_CONTENTS_BYTES,                                1e5 );
 
 	// KeyValueStoreRocksDB
+	init( ROCKSDB_SET_READ_TIMEOUT,         		    !isSimulated );
```

Comment:
Why can't we set this knob to higher value ROCKSDB_READ_VALUE_TIMEOUT may be like 10 days?

Reply:
I am not sure if that will work, but if we intend to ignore the timeout, it is better to be explicit.

Reply:
It became too complex with all these knobs is what I felt.

Reply:
This one is straightforward, no timeout in sim